lib/resourceutil/work.go

```diff
@@ -15,30 +15,31 @@ import (
 // labels[WorkPipelineNameLabel] = pipelineName
 // labels[WorkTypeLabel] = v1alpha1.WorkTypeResource
 
-func SetPromiseWorkLabels(l map[string]string, promiseName, pipelineName string) {
-	setWorkLabels(l, promiseName, "", pipelineName)
+func SetPromiseWorkLabels(l map[string]string, promiseName, pipelineName, workflowType string) {
+	setWorkLabels(l, promiseName, "", pipelineName, workflowType)
 }
 
-func SetResourceWorkLabels(l map[string]string, promiseName, resourceName, pipelineName string) {
-	setWorkLabels(l, promiseName, resourceName, pipelineName)
+func SetResourceWorkLabels(l map[string]string, promiseName, resourceName, pipelineName, workflowType string) {
```

Comment:
```suggestion
func SetResourceWorkLabels(l map[string]string, promiseName, resourceName, pipelineName string) {
```
No need for `workflowType` parameter, as far as I see.